chocolatey/choco
# (doc) Update description of page-size argument

## Description Of Changes

The behavior when specifying Page Size changed in
v2.0.0 but the description of the argument that can
be used was not updated. THis PR updates the
description of this argument.

## Motivation and Context

To have the documentation reflect the behavior of paging

## Testing

1. Run `choco search -h`
2. Ensure help screen shows the updated description of the argument

### Operating Systems Testing

- Windows 10

## Change Types Made



## Change Checklist


## Related Issue

N/A
